GameData/RealismOverhaul/RO_SuggestedMods/RealScaleBoosters/RO_RSB_Delta.cfg

```diff
@@ -746,25 +759,16 @@
     @maxTemp = 773.15
     %skinMaxTemp = 873.15
 
+    !MODULE[ModuleFuelTanks],*{}
+
     MODULE
     {
         name = ModuleFuelTanks
-        type = ServiceModule
-        volume = 58782.54
+        type = Cryogenic
```

Comment:
Considering that the DCSS has almost the same capabilities as the Centaur V then:

* We exclude any "ServiceModule", "Structural" or "Fuselage" tank types.
* Being able for long duration flights indicates that the tank type cannot be "Default".
* Being a structurally stable stage indicates that the tank type cannot be "Balloon" or "BalloonCryo".
* It is also insulated by the same insulation type as the STS ET and the SLS core stage (using foam applied by a spray machine) and i think that is what "Cryogenic" is modelled after.

The LiquidHype Delta III configs also use the same tank definition but again i did not validate if it is actually 100% correct: https://github.com/KSP-RO/RealismOverhaul/blob/master/GameData/RealismOverhaul/RO_SuggestedMods/Liquidhype/RO_LqdH_Delta3.cfg#L28